tests/apps/jsonlayerstest.cpp

```diff
@@ -396,6 +396,7 @@ static void init_frames(int32_t width, int32_t height) {
         case LAYER_TYPE_GL:
           renderer = new GLCubeLayerRenderer(gbm.dev, false);
           break;
+#ifdef USE_MINIGBM
```

Comment:
As explained my thought above, just wanted to enable only what works with GBM today to avoid making changes else which which might not yet work. But soon I will test other renderers and enable them